src/Controller/FeedbackController.php

```diff
@@ -41,7 +46,7 @@ public function beforeFilter(Event $event) {
 	 * @return \Cake\Http\Response|null
 	 */
 	public function save() {
-	    $this->request->allowMethod('post');
+	    $this->request->allowMethod(['post', 'ajax']);
```

Comment:
Isnt an ajax just a flavor? I mean usually we post+ajax, so this should be covered.
Whereas get+ajax shouldnt, right?